deeplearning/ml4pl/models/ggnn/ggnn_modules.py

```diff
@@ -211,10 +258,18 @@ def __init__(self, config, pretrained_embeddings=None):
     elif config.inst2vec_embeddings == "random":
       app.Log(1, "Initializing with random embeddings")
       self.node_embs = nn.Embedding(config.vocab_size, config.emb_size)
+    elif config.inst2vec_embeddings == "none":
+      app.Log(
+        1, "Initializing with a embedding for statements and identifiers each."
+      )
+      self.node_embs = nn.Embedding(2, config.emb_size)
     else:
       raise NotImplementedError(config.inst2vec_embeddings)
 
-    if config.use_selector_embeddings:
+    if (
+      hasattr(config, "use_selector_embeddings")
+      and config.use_selector_embeddings
+    ):
       selector_init = torch.tensor(
         # TODO(github.com/ChrisCummins/ProGraML/issues/27): x50 is maybe a
```

Comment:
I wonder if this is an issue, but please keep it in mind when restarting unrolling experiments.

Reply:
Wiill do!